### PR TITLE
Command to generate a test database, part 1

### PR DESCRIPTION
**Pull request by rowanseymour**
So far only generates non-message data: locations, orgs, users, groups, fields, labels, test contacts, contacts, contact group memberships, contact field values. Figure I'll add messages when I get onto a improving message searching performance.

The default settings generate a database with 100 orgs and 1,000,000 contacts. This takes about an hour and the resultant database is ~3.1GB.

PR includes a ~7MB dump of the AdminBoundary table in Postgres's compressed/custom format, after loading the test-data/nigeria.zip geojson file. Loading the geojson file takes half an hour but the dump loads in seconds.

### CODE REVIEW

File: temba/utils/management/commands/make_test_db.py [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+                urns = []
+                values = []
+                memberships = []
+                for c in index_batch:  # pragma: no cover
+
+                    # calculate the database id this contact will have when created
+                    c_id = base_contact_id + c
+
+                    # ensure every org gets at least one contact
+                    org = orgs[c] if c < len(orgs) else self.random_org(orgs)
```
Comment (rowanseymour) [inline temba/utils/management/commands/make_test_db.py]: This is important to ensure contacts for different orgs aren't stored sequentially which wouldn't resemble a real world database, and would effect query performance when an org's contacts are all bunched at the beginning or end of a table's data.

**Comment by nicpottier** [comment]
This is 1,000,000 contacts across 100 orgs or each has 1,000,000 contacts? Seems like the former is probably overkill. Seems our scaling issues tend to focus around single orgs being large, so wonder if we could cut down on the build speed by just building 10 orgs instead of 100.

File: temba/utils/management/commands/make_test_db.py [hunk 2]
```diff
@@ -0,0 +1,418 @@
...
+        # monkey patch uuid4 so it returns the same UUIDs for the same seed
+        from temba.utils import models
+        models.uuid4 = lambda: uuid.UUID(int=random.getrandbits(128))
+
+        self._log("Generating random test database (seed=%d)...\n" % seed)
+
+        # We want a variety of large and small orgs so when allocating content like contacts and messages, we apply a
+        # bias toward the beginning orgs. if there are N orgs, then the amount of content the first org will be
+        # allocated is (1/N) ^ (1/bias). This sets the bias so that the first org will get ~50% of the content:
+        self.org_bias = math.log(1.0 / num_orgs, 0.5)
```
Comment (nicpottier) [inline temba/utils/management/commands/make_test_db.py]: Guess I could have just read the code. :)

**Comment by rowanseymour** [comment]
It's 1,000,000 contacts split across 100 orgs. Org generation doesn't take that much time so relatively little difference between 10 and 100 orgs. Regardless of how many orgs there are, the first org always gets ~50% of the total contacts.

One thing I haven't tried yet is ditching all indexes and recreating them after, but am trying to not over-engineer things at this point.

**Comment by rowanseymour** [comment]
A sample of a first org in a database with total 1,000,000 contacts:

screen shot 2017-02-20 at 16 19 33

**Pull request merged**